Objects/typeobject.c

```diff
@@ -8759,8 +8780,15 @@ wrap_ternaryfunc(PyObject *self, PyObject *args, void *wrapped)
 
     /* Note: This wrapper only works for __pow__() */
 
-    if (!PyArg_UnpackTuple(args, "", 1, 2, &other, &third))
+    Py_ssize_t size = check_num_varargs(args, 1, 2);
```

Comment:
I think it should be safe to make `wrap_ternaryfunc` pass a hard-coded method name `"__pow__"` to `PyArg_UnpackTuple` since it is used only by `__pow__`.